README.md

```diff
@@ -87,7 +87,7 @@ Unikraft is licensed under a BSD-3-Clause.  For more information, please refer t
 [unikraft-website]: http://unikraft.org
 [unikraft-docs]: http://docs.unikraft.org
 [unikraft-guides]: https://github.com/unikraft/unikraft/tree/staging/docs/guides
```

Comment:
```suggestion
```

Reply:
I think it's better to delete this reference and rephrase the Further Resources section. I can't make a suggestion on that part but it can be something like:
"""
For more information about Unikraft, you can visit our [documentation][unikraft-docs] or check some tutorials on [unikraft-summer-of-code][https://usoc21.unikraft.org/docs/].
"""

Reply:
I've created a list of links under the subheading "Additional resources" which points to key features and concepts.